### ISSUE TITLE
Error: "Parameter 'url' must be a string

### ISSUE THREAD

**Issue by Munter** [greenkeeperio/greenkeeper#15]
I just tried enabling greenkeeper on https://github.com/Munter/tolk

```
$ greenkeeper login --public-only
$ greenkeeper --version
1.6.4
$ cd git/tolk
$ greenkeeper enable

url.js:107
    throw new TypeError("Parameter 'url' must be a string, not " + typeof url)
          ^
TypeError: Parameter 'url' must be a string, not undefined
    at Url.parse (url.js:107:11)
    at urlParse (url.js:101:5)
    at /opt/boxen/nodenv/versions/v0.10.26/lib/node_modules/greenkeeper/node_modules/github-slug/index.js:9:18
    at /opt/boxen/nodenv/versions/v0.10.26/lib/node_modules/greenkeeper/node_modules/github-slug/node_modules/git-remote-origin-url/url.js:16:3
    at /opt/boxen/nodenv/versions/v0.10.26/lib/node_modules/greenkeeper/node_modules/github-slug/node_modules/git-remote-origin-url/node_modules/gitconfiglocal/index.js:17:7
    at fs.js:266:14
    at /opt/boxen/nodenv/versions/v0.10.26/lib/node_modules/greenkeeper/node_modules/update-notifier/node_modules/configstore/node_modules/graceful-fs/graceful-fs.js:42:10
    at Object.oncomplete (fs.js:107:15)
```

**Comment by deyhle** [comment]
I also have this problem in my repositories. node@4.1.0, npm@2.14.4.

**Comment by boennemann** [comment]
This error keeps reappearing for some people, but we couldn't reproduce this yet. It seems like a third party module is making some false assumptions about the `.git/config` file.

Would anyone of you (@munter, @deyhle, @hemanth) be so kind and share their config file, so we can try and reproduce and fix this.

Best,
Stephan

**Comment by deyhle** [comment]
Maybe it occurs when there are multiple remotes specified, which are not only github…?
Here is one of my `.git/config` files (sensitive data removed):

```
[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
	precomposeunicode = true
[remote "origin"]
	url = ssh://git@stash.***************:7999/*****************.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[branch "master"]
	remote = github
	merge = refs/heads/master
[gitflow "branch"]
	master = master
	develop = develop
[gitflow "prefix"]
	feature = feature/
	release = release/
	hotfix = hotfix/
	support = support/
	versiontag = 
[branch "develop"]
	remote = github
	merge = refs/heads/develop
[remote "github"]
	url = git@github.com:****************.git
	fetch = +refs/heads/*:refs/remotes/github/*
```

**Comment by boennemann** [comment]
@deyhle Alright that's it. We're trying to read the remote "origin" and there is small bug in the module we're using. I've submitted a fix upstream: https://github.com/finnp/github-slug/pull/3

In the meantime you can use `greenkeeper enable --slug <user>/<repo>` to enable your repositories.

Sorry for the inconvenience.

Best,
Stephan

**Comment by boennemann** [comment]
Alright, thanks for your reports and help everyone. A fix landed as of 2.0.1 https://github.com/greenkeeperio/greenkeeper/commit/e20a79c499b2f1408b0b28854270ac880df35d15

Best,
Stephan
